### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Hedy/Adventures](https://hosted.weblate.org/projects/hedy/adventures/).

It also includes following components:

* [Hedy/Achievements](https://hosted.weblate.org/projects/hedy/achievements/)

* [Hedy/Webpages](https://hosted.weblate.org/projects/hedy/webpages/)

* [Hedy/slides](https://hosted.weblate.org/projects/hedy/slides/)

* [Hedy/Parsons](https://hosted.weblate.org/projects/hedy/parsons/)

* [Hedy/client-messages](https://hosted.weblate.org/projects/hedy/client-messages/)

* [Hedy/Keywords](https://hosted.weblate.org/projects/hedy/keywords/)

* [Hedy/Cheatsheets](https://hosted.weblate.org/projects/hedy/commands/)

* [Hedy/Quizzes](https://hosted.weblate.org/projects/hedy/quizzes/)

* [Hedy/Texts](https://hosted.weblate.org/projects/hedy/web-texts/)

* [Hedy/Tutorials](https://hosted.weblate.org/projects/hedy/tutorials/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by Felienne** [comment]
Hmmm @jpelay, it seems weblate is not 100% happy, in id and es some lines seem to be unwrapped again. I am not sure why 😭

**Comment by jpelay** [comment]
> Hmmm @jpelay, it seems weblate is not 100% happy, in id and es some lines seem to be unwrapped again. I am not sure why 😭

Yes, I saw! Apparently setting the add-on to 'No line wrapping' is not doing the work, and we need to set it to  'Wrap on new lines' which they say is like `--no-wrap`.

File: translations/es/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -3,108 +3,168 @@
...
+"¡Oops! Has comenzado una línea con un espacio en la línea {line_number}. Los "
+"espacios confunden a los ordenadores, ¿podrías quitarlos?"
 
 msgid "Invalid Type Combination"
-msgstr "No puedes usar {invalid_argument} y {invalid_argument_2} con {command} porque uno es {invalid_type} y el otro es {invalid_type_2}. Intenta cambiar {invalid_argument} a {invalid_type_2} o {invalid_argument_2} a {invalid_type}."
+msgstr ""
+"No puedes usar {invalid_argument} y {invalid_argument_2} con {command} "
+"porque uno es {invalid_type} y el otro es {invalid_type_2}. Intenta cambiar "
+"{invalid_argument} a {invalid_type_2} o {invalid_argument_2} a "
+"{invalid_type}."
```
Comment (jpelay) [inline translations/es/LC_MESSAGES/messages.po:78]: Still the same problem :/

File: translations/es/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -3,108 +3,168 @@
...
 
 msgid "Missing Inner Command"
-msgstr "Parece que has olvidad usar un comando en el {command} que has usado en la línea {line_number}."
+msgstr ""
+"Parece que has olvidad usar un comando en el {command} que has usado en la "
+"línea {line_number}."
 
 msgid "Misspelled At Command"
-msgstr ""
+msgstr "Parece que haz escrito incorrectamente el comando {command}, en su lugar escribiste {invalid_argument} en la línea {line_number}."
```
Comment (jpelay) [inline translations/es/LC_MESSAGES/messages.po:104]: Although, I just translated this one and it wasn't wrapped, so maybe we solved the problem, but Weblate only runs --no-wrap on labels that were edited?

Comment (jpelay) [reply translations/es/LC_MESSAGES/messages.po:104]: Someone that speaks a language other than Spanish, can please translate some text in other language to see what happens (@Mark-Giesen @Felienne)

File: translations/nl/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -67,9 +67,8 @@ msgstr "Het lijkt erop dat je vergeten bent om een commando te gebruiken op rege
 msgid "Missing Inner Command"
 msgstr "Het lijkt erop dat je vergeten bent om het {command} commando op regel {line_number} in te vullen."
 
-#, fuzzy
 msgid "Misspelled At Command"
-msgstr "It looks like you might have misspelled the {command} command, instead you wrote {invalid_argument} in line {line_number}."
+msgstr "Je hebt {command} verkeerd geschreven, jij schreef {invalid_argument} op regel {line_number}."
```
Comment (jpelay) [inline translations/nl/LC_MESSAGES/messages.po:71]: It seems like the change I did to the plug-in did work properly. I'm afraid we'll have to run the --no-wrap for es and id ourselves, but hopefully in the future we won't have as many changes!

Comment (Felienne) [reply translations/nl/LC_MESSAGES/messages.po:71]: Awesome!! Feel free to approve this so we can than do another unwrapping one (and update the Wiki, again?)

Comment (jpelay) [reply translations/nl/LC_MESSAGES/messages.po:71]: Wiki updated!

**Review by jpelay — APPROVED** [review]

**Pull request merged**